revpi-hat-eep/src/bin/revpi-eep.rs

```diff
@@ -184,6 +154,34 @@ pub struct Cli {
     pub outfile_name: PathBuf,
 }
 
+fn export_config(config: &RevPiHatEeprom, backup: PathBuf) {
+    let json = serde_json::to_string(config)
+        .expect("BUG: Can't create (full) json from RevPiHatEeprom config");
+    let mut backup_file = match OpenOptions::new()
+        .read(false)
+        .write(true)
+        .truncate(true)
+        .create(true)
+        .open(&backup)
+    {
+        Ok(file) => file,
+        Err(e) => {
+            eprintln!(
+                "ERROR: Can't open json backup file: `{}': {e}",
+                backup.to_string_lossy()
+            );
+            process::exit(1);
+        }
+    };
+    if let Err(e) = backup_file.write_all(json.as_bytes()) {
```

Comment:
backup_file is the exported json? Would it call json_export_file or something like that. The function is also called export_config.